src/Dapr.Actors.AspNetCore/ActorsServiceCollectionExtensions.cs

```diff
@@ -45,6 +46,16 @@ public static void AddActors(this IServiceCollection services, Action<ActorRunti
             services.TryAddSingleton<ActorRuntime>(s =>
             {   
                 var options = s.GetRequiredService<IOptions<ActorRuntimeOptions>>().Value;
+
+                //Replace the HttpEndpoint with an endpoint prioritizing IConfiguration
+                var configuration = s.GetService<IConfiguration>();
```

Comment:
Should we do required service here ?

Reply:
Nope - I wanted to leave the door open here for anyone that isn't fully bought into DI for whatever reason (e.g. read: ensure this isn't a breaking change) so that if IConfiguration somehow isn't populated, this isn't going to suddenly break their apps.